tests/testBackgroundsClass.m

```diff
@@ -132,14 +135,14 @@ function testSetBackground(testCase)
 
          testCase.background.setBackground(1, 'type', allowedTypes.Constant.value);
          testCase.verifyEqual(testCase.background.backgrounds.varTable{1, 2}, string(allowedTypes.Constant.value), 'setBackground method not working');
-         testCase.background.setBackground('Background 1', 'type', allowedTypes.Function); 
-         testCase.verifyEqual(testCase.background.backgrounds.varTable{1, 2}, string(allowedTypes.Function.value), 'setBackground method not working');
-         testCase.verifyError(@() testCase.background.setBackground(2, 'type', 'random'), exceptions.invalidOption.errorID);
+         %testCase.background.setBackground('Background 1', 'type', allowedTypes.Function); 
+         %testCase.verifyEqual(testCase.background.backgrounds.varTable{1, 2}, string(allowedTypes.Function.value), 'setBackground method not working');
+         %testCase.verifyError(@() testCase.background.setBackground(2, 'type', 'random'), exceptions.invalidOption.errorID);
          
-         testCase.background.setBackground(3, 'Value1', 'random');
-         testCase.verifyEqual(testCase.background.backgrounds.varTable{3, 3}, "random", 'setBackground method not working');
-         testCase.background.setBackground('Background 3', 'Value1', 'Background param 1');
-         testCase.verifyEqual(testCase.background.backgrounds.varTable{3, 3}, "Background param 1", 'setBackground method not working');
+         %testCase.background.setBackground(3, 'Value1', 'random');
+         %testCase.verifyEqual(testCase.background.backgrounds.varTable{3, 3}, "random", 'setBackground method not working');
+         %testCase.background.setBackground('Background 3', 'Value1', 'Background param 1');
+         %testCase.verifyEqual(testCase.background.backgrounds.varTable{3, 3}, "Background param 1", 'setBackground method not working');
```

Comment:
and these lines.

Reply:
can't uncomment 142-143 as i'd changed background 3 to a constant background - Value1 of a constant background is validated but Value1 of a function background is not, so you can't just arbitrarily set Value1 for a test unless the background type is function